tools/project_manager/add_files.php

```diff
@@ -835,108 +818,61 @@ public function execute(): void
             $src_text_file_path = "$this->source_project_dir/$src_text_file_name";
 
             if ($text_a == 'add' && $image_a == 'add') {
-                if ($this->dry_run) {
-                    echo "
-                        project_add_page(
-                            $this->projectid,
-                            $base,
-                            $src_image_file_name,
-                            $src_text_file_path,
-                            $pguser,
-                            $now );<br>
-                    ";
-                } else {
-                    try {
-                        project_add_page(
-                            $this->projectid,
-                            $base,
-                            $src_image_file_name,
-                            $src_text_file_path,
-                            $pguser,
-                            $now
-                        );
-                    } catch (DBQueryError $error) {
-                        echo "<p class='error'>";
-                        echo "for base=$base, project_add_page raised a DB error";
-                        echo "</p>";
-                    }
+                try {
+                    project_add_page(
+                        $this->projectid,
+                        $base,
+                        $src_image_file_name,
+                        $src_text_file_path,
+                        $pguser,
+                        $now
+                    );
+                } catch (DBQueryError $error) {
+                    echo "<p class='error'>";
+                    echo "for base=$base, project_add_page raised a DB error";
+                    echo "</p>";
```

Comment:
Nit: This could use the new `error` helper:

```suggestion
                    $this->error("for base=$base, project_add_page raised a DB error");
```